src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -1,14 +1,26 @@
 import React, { KeyboardEvent, ReactNode, useCallback, useMemo } from 'react';
 
+import { defineMessages } from 'react-intl';
 import UserMention from '~core/UserMention';
 import { ListGroupItem } from '~core/ListGroup';
-import { AnyUser } from '~data/index';
+import { AnyUser, useUserReputationQuery } from '~data/index';
 import { Address, ENTER } from '~types/index';
 import HookedUserAvatar from '~users/HookedUserAvatar';
 import { getMainClasses } from '~utils/css';
+import MaskedAddress from '~core/MaskedAddress';
+import Numeral from '~core/Numeral';
+import { useTokenInfo } from '~utils/hooks/use-token-info';
+import Icon from '~core/Icon';
 
 import styles from './MembersListItem.css';
 
+const MSG = defineMessages({
+  starReputationTitle: {
+    id: 'MembersList.MembersListItem.starReputationTitle',
```

Comment:
This id should be in the `core` namespace:
```suggestion
    id: 'core.MembersList.MembersListItem.starReputationTitle',
```